privacyidea/app.py

```diff
@@ -196,38 +171,37 @@ def create_app(config_name="development",
 
     app.response_class = PiResponseClass
 
+    # Setup logging
     try:
         # Try to read logging config from file
         log_config_file = app.config.get("PI_LOGCONFIG",
                                          "/etc/privacyidea/logging.cfg")
         if os.path.isfile(log_config_file):
-            logging.config.fileConfig(log_config_file)
+            _, ext = os.path.splitext(log_config_file)
+            if ext.upper() in ['.YML', '.YAML']:
```

Comment:
Honestly I do not like file endings so much! :-/ Sounds like windows.
I would think it is more pythonic to catch eceptions.
Or to use something like python-magic. On the other hand, I do not want to get it too bloated.
What was your reason to go for file endings?

Reply:
Unfortunately there is no way in detecting the content type of the file (except reading it and catching an exception).
Both the builtin mimetypes and python-magic only detect it as plain text files.

Reply:
Fixed in 9136a076

Reply:
thanks - looks awesome! :-)

Reply:
Fixed typo and docu in 42218553